mobile/src/home-view.js

```diff
@@ -126,6 +125,14 @@ export default class HomeView extends PureComponent {
       </View>
     )
   }
+
+  returnIsDisplayable = (s, sessions) => {
+    var isDisplayable = false
+    const currentTitle = s.name
+    const dup = Object.values(sessions).find(i => i.name.toLowerCase() === currentTitle.toLowerCase())
+    if (currentTitle.length && !dup) { isDisplayable = true } 
```

Comment:
Instead of initializing `isDisplayable` to false, then possibly setting to true, just say `const isDisplayable = currentTitle.length > 0 && !dup`